athena/llm_core/llm_core/models/__init__.py

```
@@ -1,47 +1,18 @@
-type0 = types[0]
-if len(types) == 1:
-    ModelConfigType = type0
-else:
-    type1 = types[1]
-    ModelConfigType = Union[type0, type1] # type: ignore
-    
\ No newline at end of file
+__all__ = [
+    "ModelConfig",
+    "ModelConfigType",
+    "BaseChatModelConfig",
```

Pylint (3.3.7)

[error] Undefined variable name 'BaseChatModelConfig' in __all__

(E0603)